Dockerfile

```diff
@@ -10,7 +10,7 @@ COPY . .
 
 RUN CGO_ENABLED=0 GOOS=linux go build -a -tags netgo -ldflags '-w -extldflags "-static"' -o /usr/local/bin/checkout main.go
 
-FROM alpine:3.19
+FROM alpine:3.20.0
```

Comment:
Generally, I'd not pin patch versions but the minor version only, e.g. `alpine:3.20`. This is so that we receive security updates implicitly every time we rebuild the image (without cache) for whatever reason as opposed to explicitly having to change the code for every single security update.
Not blocking the PR really, though.
fyi: When image references within a Dockerfile are pulled, there is no caching involved but the entire base image is reloaded from scratch during every build - as opposed to image references within the `uses` fields of our workflows where currently it may make sense to pin the patch version because those images are cached and therefore without specifying the patch version, we may never receive a security update or only within some builds, depending on the k8s node the build runs on.